Xamarin.Essentials/Calendars/Calendars.android.cs

```diff
@@ -622,7 +624,8 @@ static RecurrenceRule GetRecurranceRuleForEvent(string rule)
             {
                 var ruleOccurenceMonthDays = rule.Substring(rule.IndexOf("BYMONTHDAY=", StringComparison.Ordinal) + 11);
                 ruleOccurenceMonthDays = ruleOccurenceMonthDays.Contains(";") ? ruleOccurenceMonthDays.Substring(0, ruleOccurenceMonthDays.IndexOf(";", StringComparison.Ordinal)) : ruleOccurenceMonthDays;
```

Comment:
Can ruleOccurenceMonthDays be null?

Reply:
not that I'm aware of, Substring could return a string.Empty but I wouldn't have thought it could return null?

Reply:
ah yes, more of a question of whether "rule" could be null, in which case it should be rule?.Substring... which means that ruleOccurrenceMonthDays could be null. 
Also, what's the magic number 11 in the previous row and can we extract "BYMONTHDAY" as a string constant

Reply:
Yeah I'll create a constant for BYMONTHDAY= and then i can use that const's .length instead of 11

(and apply this change + ? for the other similar calls too)

Reply:
The only question I have is wouldn't it be impossible for rule to be null if it passes the condition of:
if (rule.Contains(byMonthDaySearch))